client/src/ratings_reviews/ReviewTile.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+let ReviewTile = (props) => {
```

Comment:
change `let ReviewTile` to `const ReviewTile`

Reply:
Change made!